src/cluster_legacy.c

```diff
@@ -546,7 +546,7 @@ int clusterLoadConfig(char *filename) {
             } else if (!strcasecmp(s, "noflags")) {
                 /* nothing to do */
             } else {
-                serverPanic("Unknown flag in redis cluster config file");
+                serverPanic("Unknown flag in %s cluster config file", SERVER_NAME);
```

Comment:
Better to use SERVER_TITLE here.

SERVER_TITLE = "Valkey" (uppercase "V")
SERVER_NAME = "valkey" (lowercase "v")

```suggestion
                serverPanic("Unknown flag in %s cluster config file", SERVER_TITLE);
```